From 733a85c14cb4ade510a0c78b32cd12cb2d70dbbd Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Sat, 22 Feb 2014 17:47:44 +0100
Subject: [PATCH] examples/wayland: include server-buffer only when building
 with opengles2

* it's using glBindBuffer in
  server-buffer/client/serverbufferrenderer.cpp

Change-Id: I8412dfd4ebb95c147328ac6e4dfff14a0cff4e78
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 examples/wayland/wayland.pro | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/examples/wayland/wayland.pro b/examples/wayland/wayland.pro
index 503190c..9943c52 100644
--- a/examples/wayland/wayland.pro
+++ b/examples/wayland/wayland.pro
@@ -10,4 +10,6 @@ qtHaveModule(quick) {
     SUBDIRS += custom-extension
 }
 
-SUBDIRS += server-buffer
+contains(QT_CONFIG, opengles2) {
+    SUBDIRS += server-buffer
+}
